docs/protocol/specifications/blockchain.md

```diff
@@ -0,0 +1,613 @@
+
+Witness field       | Type                       | Description
+--------------------|----------------------------|----------------
+Destinations        | List<ValueDestination1>    | The Destinations ("forward pointers") for the value contained in this Mux. This can point directly to Output entries, or to other Muxes, which point to Output entries via their own Destinations.
+Arguments           | String                     | Arguments for the program contained in the Nonce.
+
+#### Mux Validation
+
+1. [Validate](#program-validation) `Program` with the given `Arguments` and the transaction version.
+2. For each `Source` in `Sources`, [validate](#value-source-1-validation) `Source`.
+3. For each `Destination` in `Destinations`, [validate](#value-destination-1-validation) `Destination`.
+4. For each `AssetID` represented in `Sources` and `Destinations`:
+    1. Sum the total `Amounts` of the `Sources` with that asset ID. Validation fails if the sum overflows 63-bit integer.
+    2. Sum the total `Amounts` of the `Destinations` with that asset ID. Validation fails if the sum overflows 63-bit integer.
+    3. Verify that the two sums are equal.
+5. Verify that for every asset ID among `Destinations`, there is at least one `Source` with such asset ID. (This prevents creating zero units of an asset not present among the valid sources.)
+6. If the transaction version is 1: verify that the `ExtHash` is the all-zero hash.
+
+
+
+### Nonce
+
+Field               | Type                 | Description
+--------------------|----------------------|----------------
+Type                | String               | "nonce"
+Body                | Struct               | See below.
+Witness             | Struct               | See below.
+
+Body field          | Type                 | Description
+--------------------|----------------------|----------------
+Program             | Program              | A program that protects the nonce against replay and must evaluate to true.
+Time Range          | Pointer<TimeRange>   | Reference to a TimeRange entry.
+ExtHash             | [ExtStruct](#extension-struct) | If the transaction version is known, this must be 32 zero-bytes.
+
+Witness field       | Type                         | Description
+--------------------|------------------------------|----------------
+Arguments           | List<String>                 | Arguments for the program contained in the Nonce.
+AnchoredEntry       | Pointer                      | Optional pointer to a single entry of any type, which uniquely identifies that entry as one that can use this one as an `Anchor`.
+
+
+#### Nonce Validation
+
+1. [Validate](#program-validation) `Program` with the given `Arguments`.
+2. Verify that `Issuance` points to an issuance that is present in the transaction (meaning visitable by traversing `Results` and `Sources` from the transaction header) and whose `Anchor` is equal to this nonce's ID.
```

Comment:
Yes, good catch—this rule should actually be eliminated. The `AnchoredEntry` should only be validated by the entry pointing to this one.